spacy_llm/tasks/util/span.py

```diff
@@ -12,6 +14,15 @@
 class SpanTask:
     """Base class for Span-related tasks, eg NER and SpanCat."""
 
+    PLAIN_CONFIG_KEYS: List[str] = [
```

Comment:
Assuming this shouldn't be accessed externally (also for other occurrences).
```suggestion
    _PLAIN_CONFIG_KEYS: List[str] = [
```

Reply:
👍